doc/manual/citing_aspect.bib

```diff
@@ -1203,3 +1203,32 @@ @Article{se-13-849-2022
 URL = {https://se.copernicus.org/articles/13/849/2022/},
 DOI = {10.5194/se-13-849-2022}
 }
+
+@mastersthesis { quiroga2022numericalm,
+	title = {Numerical Models of Lithosphere Removal in the Sierra Nevada de Santa Marta, Colombia},
```

Comment:
there seems to be a missing ``author`` here and below.

Reply:
Wow, that should not have got by me but did point out that the export feature on the website does not work so well.